worker/docker_client.py

```diff
@@ -40,10 +40,15 @@ class DockerClient(object):
     """
     Methods for talking to Docker.
```

Comment:
Add some more high-level documentation on trying to use nvidia-docker and what our docker solution is.  Say something about trying to use GPUs.

Separately, on the Wiki, we should add some instructions on how to install/use nvidia-docker.